controllers/orderController.js

```diff
@@ -0,0 +1,42 @@
+const orderService = require('../services/orderService');
+const { catchAsync } = require('../utils/error');
+
+const createOrder = catchAsync(async( req, res )=> {
+  const  userId  = req.user.userId;
+
+  if( !req.body.roomId || !req.body.checkIn || !req.body.checkOut){
+    const error = new Error('CHECK_YOUR_BODY');
+    error.statusCode = 401;
+    throw error;
+  }
+  const { roomId, checkIn, checkOut } = req.body;
```

Comment:
12번 코드 라인을 통해 request의 body에서 3개 정보를 구조분해할당을 통해 값을 받아오고 있습니다.

이에 따른 예외처리를 진행해주세요!

에러메시지는 좀 더 명확하게 작성해주세요!
```suggestion
  const { roomId, checkIn, checkOut } = req.body;
  
  if( !roomId || !checkIn || !checkOut){
    const error = new Error('KEY_ERROR');
    error.statusCode = 400;
    throw error;
  }

```